src/libs/AxelarQueryAPI.ts

```diff
@@ -381,16 +384,21 @@ export class AxelarQueryAPI {
 
     // verify asset params
     if (!this.allAssets) await this._initializeAssets();
-    const asset = this.allAssets.find((asset) => asset.common_key[this.environment] === denom);
-    if (!asset) throw `Asset ${denom} not found`;
+    const assetConfig = this.allAssets.find(
+      (asset) => asset.common_key[this.environment] === denom.toLowerCase()
+    );
+    if (!assetConfig) throw `Asset ${denom} not found`;
```

Comment:
This now becomes a duplication code that checks if the denom exists. Because the new `_convertAssetDenom` function also did the check. So, I think we can replace all the checks here with the `_convertAssetDenom` function and remove the `this._convertAssetDenom` usage below.

Reply:
makes sense to me

Reply:
done